src/pyedb/dotnet/database/cell/terminal/terminal.py

```diff
@@ -195,6 +195,15 @@ def is_reference_terminal(self):
         """Whether it is a reference terminal."""
         return self._edb_object.IsReferenceTerminal()
 
+    @property
+    def reference_terminal(self):
+        """Adding grpc compatibility."""
+        return self.ref_terminal
+
+    @reference_terminal.setter
+    def reference_terminal(self, value):
+        self.ref_terminal = value
+
     @property
     def ref_terminal(self):
```

Comment:
can you add deprecation on this property as Massimo suggests?

Reply:
Done thanks for pointing this out.